src/main/java/edu/ucsb/cs156/organic/controllers/CoursesController.java

```diff
@@ -195,5 +198,23 @@ public Course updateCourse(
         return course;
     }
 
-    
+    @Transactional
```

Comment:
In response to the question about whether this is the right way to make this a transaction (i.e. either all of the SQL queries succeed, or none of them do), the simple answer is: I'm not sure.

I found this: https://www.baeldung.com/transaction-configuration-with-jpa-and-spring

I'm going to assume that this is correct unless/until we discover otherwise.

Reply:
Agreed, I came across an article of the same website describing how to use the derived delete methods of Spring Boot: https://www.baeldung.com/spring-data-jpa-deleteby
Here they use Transactional too.